template/src/hooks/ColorHook/index.js

```diff
@@ -0,0 +1,10 @@
+import {useGlobalState} from 'tram-one'
+
+const APP_COLOR = 'appColor'
```

Comment:
It's a bit of nit-pick, but I believe standard convention is to make the value all caps with underscore (like the variable name), so it should look like `APP_COLOR`